css-route-matching/index.bs

```diff
@@ -122,42 +128,61 @@ as follows:
 	: from: <<route-location>>
 	:: The result is true if
 		the [=document's navigation API=] of the document
-		is non-null,
-		its [=transition=] is non-null,
-		its [=from entry=]'s {{NavigationHistoryEntry/url}}
-		is non-null and
-		[=URL pattern/match|matches=]
-		the [=route location URL pattern=] of <<route-location>>.
+		is non-null, and either:
+
+		* its [=transition=] is non-null,
+			its [=from entry=]'s {{NavigationHistoryEntry/url}}
+			is non-null and
+			[=URL pattern/match|matches=]
+			the [=route location URL pattern=] of <<route-location>>.
+
+		* its [=activation=] is non-null,
+			the document's [=has been revealed=] is false or
+			the [=reveal a document=] steps that began when
+			[=has been revealed=] was false are still running,
+			and the activation's {{NavigationActivation/from}}'s
+			{{NavigationHistoryEntry/url}}'s
+			is non-null and
+			[=URL pattern/match|matches=]
+			the [=route location URL pattern=] of <<route-location>>.
 
 	: to: <<route-location>>
 	:: The result is true if
 		the [=document's navigation API=] of the document
-		is non-null,
-		its [=ongoing navigate event=] is non-null,
-		and its {{NavigateEvent/destination}}'s
-		{{NavigationDestination/url}}
-		[=URL pattern/match|matches=]
-		the [=route location URL pattern=] of <<route-location>>.
-
-		<!--
-
-		After
-		<a href="https://github.com/whatwg/html/issues/11690">whatwg/html#11690</a> /
-		<a href="https://github.com/whatwg/html/pull/11692">whatwg/html#11692</a>.
-		we could probably define this more like "from" above.  But it's probably
-		also OK as is since the transition and the ongoing navigate event basically have
-		the same lifetime.
-
-		-->
-
-	ISSUE: The above definitions of from and to probably don't do the right thing
-	during the loading of the new document in a cross-document navigation,
-	since I think they reference concepts that are only present on the old document.
+		is non-null, and either:
+
+		* its [=ongoing navigate event=] is non-null,
+			and its {{NavigateEvent/destination}}'s
+			{{NavigationDestination/url}}
+			[=URL pattern/match|matches=]
+			the [=route location URL pattern=] of <<route-location>>.
+
+			ISSUE: This assumes that the [=ongoing navigate event=]
+			and the [=transition=] have the same lifetime,
+			but this isn't really
+			true if the event is intercepted.
+			After
+			<a href="https://github.com/whatwg/html/issues/11690">whatwg/html#11690</a> /
+			<a href="https://github.com/whatwg/html/pull/11692">whatwg/html#11692</a>.
+			we could probably define this more like "from" above.
+			But which lifetime is the one we want?
+
+		* its [=activation=] is non-null,
```

Comment:
Same as earlier.
Perhaps there should be a "document is revealing" boolean that we set at https://html.spec.whatwg.org/multipage/browsing-the-web.html#reveal and properly clean up before the end of the "update the rendering" steps.